exl_snap_backup

```diff
@@ -18,18 +18,23 @@ APP_VGNAME=vg_voyager
 APP_LVNAME=`lvs | grep ${APP_VGNAME} | awk '{ print $1 }' | egrep -v "backup|snap"`
 
 TODAY=`date +%Y%m%d`
+DAYSAGO=`date -d 'now - 1 days' "+%Y%m%d"`
 
 for LV in ${APP_LVNAME} ; do
+  # Example BACKUPDIR: /voyagerbackup/m1
   BACKUPDIR=${BASEBACKUPDIR}/`echo ${LV} | sed s/lv_//g`
 
-  # Find and remove any old/lingering back-up directories in ${BACKUPDIR}
-  # These should have already been captured by NetBackup
-  REMOVEDIR=`find ${BACKUPDIR}/* -maxdepth 0 -type d -mmin +1440 2> /dev/null`
-  if [ -n ${REMOVEDIR} ] ; then
-    for DIR in ${REMOVEDIR} ; do
-      rm -rf ${DIR}
-    done
-  fi
+  # Remove old back-up directories in ${BACKUPDIR}
+  # Keep only yesterday and today's backups
+  # All previous backups have already been captured by NetBackup
+  DIRLIST=`find ${BACKUPDIR}/* -maxdepth 0 -type d -name "lv_*_snap_????????" -printf "%f\n"`
+  for DIR in ${DIRLIST} ; do
+    # Grab just the date suffix (YYYYMMDD) from the backup dir name lv_*_snap_YYYYMMDD
+    DATE=`echo ${DIR} | awk -F "_" '{ print $4 }'`
+    if [ $(( ${DATE} - ${DAYSAGO} )) -lt 0 ] ; then
```

Comment:
The logic and syntax are fine, but see my previous comment.  I would rename `DAYSAGO` to `CUTOFF_DATE` or `OLDEST_DATE` or something which better represents its format and intent.

Other than that, all fine with me.